hw/ip/rv_timer/dv/rv_timer_sim_cfg.hjson

```diff
@@ -46,7 +46,14 @@
   tests: [
     {
       name: rv_timer_smoke
-      uvm_test_seq: rv_timer_smoke_vseq
+      uvm_test_seq: rv_timer_random_vseq
+      run_opts: ["+smoke_test=1"]
+      reseed: 1
```

Comment:
nit: smoke is often a simple version of `random` test, still good to have some reseeds to catch issues introduced accidentally in smoke. smoke also randomizes most of the setting, but it's easier to debug than the random test.

Reply:
Hmm, sure. Changed it to 20.